pkg/detectors/replicate/replicate.go

```diff
@@ -0,0 +1,85 @@
+package replicate
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct {
+	client *http.Client
+}
+
+// Ensure the Scanner satisfies the interface at compile time.
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	defaultClient = common.SaneHttpClient()
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"replicate"}) + `\b(r8_[0-9A-Za-z-_]{37})\b`)
+)
+
+func (s Scanner) Keywords() []string {
+	return []string{"replicate"}
```

Comment:
`r8_` should also be a prefix, since that's the fixes prefix of their api tokens.